soscan/utils.py

```diff
@@ -39,3 +41,180 @@ def parseDatetimeString(ds):
     if isinstance(ds, bytes):
         ds = ds.decode("utf-8")
     return dateparser.parse(ds, settings={"RETURN_AS_TIMEZONE_AWARE": True})
+
+class GeoBox(object):
```

Comment:
How are regions crossing the anti-meridian handled?

Reply:
@datadavev I just realized I forgot this. I'm adding the following:

```python
        if abs(south) > 90 or abs(north) > 90 or abs(west) > 180 or abs(east) > 180:
            self.L.warning(f"Bounding box exceeds valid bounds. SWNE: {south} {west} {north} {east}")
            return None
```